include/real/exact_number.hpp

```diff
@@ -168,7 +170,7 @@ namespace boost {
             }
 
             //Returns (a*b)/mod
-            inline T mult_div(T a, T b, T c) {
+            static inline T mult_div(T a, T b, T c) {
```

Comment:
All functions are inlined in a class by default.
https://en.cppreference.com/w/cpp/language/inline

Reply:
removed inline.